### DISCUSSION TITLE
Easy Effects doesn't start on Linux Mint 21

### DISCUSSION THREAD

**Discussion by Th0se** [wwmm/easyeffects#1962] [General]
When launched it says, 'Starting Easy Effects,' then just vanished. What exactly is happening, and is there a known solution?

**Comment by wwmm** [comment]
Try to do the step `number 3` described in our Basic Logs section https://github.com/wwmm/easyeffects/wiki/Reporting-Bugs#basic-logs. This way we will be able to see the debug messages.

**Reply by Th0se** [reply]
I get no output from that, unfortunately, even after hours of waiting. Would the output of `pw-dump` be useful?

**Reply by Th0se** [reply]
I uninstalled Easy Effects and installed Pulse Effects, and it's working so far.

**Reply by wwmm** [reply]
>I get no output from that, unfortunately, even after hours of waiting.

There should be a huge number of lines. What command did you execute exactly? Did you kill EaysEffects first?
